src/test/java/com/salesforce/einsteinbot/sdk/examples/SdkExamples.java

```diff
@@ -1,150 +0,0 @@
-/*
```

Comment:
I realized that we have `ChatbotClientExamples` and `OAuthExamples` in this package as well. Do we need them? Or can we delete them like this one?

Reply:
`ChatbotClientExamples` and `OAuthExamples` are useful. I run them after refactoring as well. Maybe we can delete them in the future once we add full working examples in public documentation.